src/vs/workbench/contrib/comments/browser/media/review.css

```diff
@@ -467,7 +467,8 @@ div.preview.inline .monaco-editor .comment-range-glyph {
 	background: var(--vscode-editorGutter-commentRangeForeground);
 }
 
-.monaco-editor .comment-thread:before {
+.monaco-editor .comment-thread:before,
+.monaco-editor .comment-thread-unresolved:before {
```

Comment:
FYI, this was the only functional change. The other changes I made were just nits where I moved some code around.

Reply:
Thanks for tidying it up.